### PR TITLE
feat(content): Split the Quarg Government

### PR DESCRIPTION
**Pull request by Saugia**
**Content (Artwork / Missions / Jobs)**

## Summary
There has been a desire to split the Quarg into more area-specific governments mentioned by @Arachi-Lover , @Amazinite , and other content creators for future use, and to work around use for area-specific conditions, fixes, etc.. This also supports the idea that in the future, we'll have a Coalition Quarg for that related content instead of messing up with the human Quarg, etc.

So far, I've added several location-related Quarg governments, while keeping the original the human Quarg. Each locational Quarg has their own attitudes with the relevant local factions.

All this said, @Arachi-Lover , this means for Coalition content you can make a Quarg (Coalition) gov to use for them, and we can have future Quarg govs separate for newer races.

Something we can keep in mind for a future PR is having different Quarg hails, languages, etc. with different mannerisms for each place. 

## Save File
N/A

### CODE REVIEW

**Comment by Arachi-Lover** [comment]
As I said on the Discord, while I do believe the Quarg are more focused on and worried about their regional species and the stuff going on there, the Quarg are pretty solidly unified as a species, so while most Quarg from other areas might not join up to fight the Heliarchs, they definetly sympathize with those Quarg who lost their homes in the 3 rings, and aren't all that fond of the Heliarchs due to all of that.
It has always been intentional, as I see it, that attacking any Quarg makes all the Quarg mad at you, and this'd apply to the Coalition case as well; I've made mention of this many times, that the idea is that if you side with the Heliarchs and win, the Quarg everywhere would see you as an enemy and be hostile. It is a pretty impactful and dire consequence, yes, but given the scope of a T2.5 campaign that directly deals with the Quarg, I feel that's only to be expected, and very appropriate. One *should* think well before making them an enemy.
To reiterate what I suggested on the Discord, I think that for more region-specific Quarg becoming mad (such as the whole thing with making them protect the Efreti more activelly and have that impact your reputation) would be best if even if it's only the Quarg there in the specific region that are openly hostile, Quarg elsewhere'd also of course learn of what you've done, and would restrict your landing as you're a known problem.

**Review by Arachi-Lover — APPROVED** [review]

File: data/governments.txt [hunk 1]
```diff
@@ -1017,25 +1020,106 @@ government "Pug (Wanderer)"
...
 	"player reputation" 1
 	"attitude toward"
+		"Quarg (Hai)" 1
+		"Quarg (Kor Efret)" 1
+		"Quarg (Gegno)" 1
 		"Merchant" .01
-		"Kor Efret" .01
-		"Kor Mereti" -.01
-		"Kor Sestor" -.01
 		"Hai" .01
```
Comment (warp-core) [inline data/governments.txt:1042]: Why are you removing these?

Comment (Saugia) [reply data/governments.txt:1042]: Do the human Quarg ever come into contact with the Automata? I forget.

Comment (warp-core) [reply data/governments.txt:1042]: This is under the `Quarg` government, and that's the government of every existing Quarg ship, including ones created in missions that are specifically intended to fight the automata (the ships they send to help the Wanderers defend Kaliptari, the Quarg ships that assist in the mission where you defend the Efreti from a Mereti attack, and the Quarg that assist in the defence of Alnitak, off the top of my head).

Even if you update those missions to use the sub-governments, this government still needs to maintain continuity for the ships that already exist.

Comment (Saugia) [reply data/governments.txt:1042]: By ships that already exist, do you mean fleets, or ones active in missions? I also updated systems with fleets that use the specific governments. But yeah, good point.

**Comment by Saugia** [comment]
>To reiterate what I suggested on the Discord, I think that for more region-specific Quarg becoming mad (such as the whole thing with making them protect the Efreti more activelly and have that impact your reputation) would be best if even if it's only the Quarg there in the specific region that are openly hostile, Quarg elsewhere'd also of course learn of what you've done, and would restrict your landing as you're a known problem.

In that case, it sounds like we might benefit from having some sort of event or tracker mission that activates once you're a certain rep with a Quarg gov, during cases such as: Upsetting the Efret Quarg, the Gegno Quarg, future Coalition Quarg, etc..

**Comment by Arachi-Lover** [comment]
Yes, kinda like how the Wanderers don't let you land anywhere but on the capital (but in this case it'd be more like, everywhere).

File: data/governments.txt [hunk 2]
```diff
@@ -1036,6 +1046,83 @@ government "Quarg"
...
+	swizzle 0
+	color .88 .77 0
+	"display name" "Quarg"
+
+	"player reputation" 1
+	"attitude toward"
+		"Quarg" 1
+		"Quarg (Hai)" 1
+		"Quarg (Gegno)" 1
+		"Kor Efret" .01
```
Comment (Hurleveur) [inline data/governments.txt:1100]: Shoudnt these Quarg care for the Efret in the same way gegno quarg care about gegno?
Mb tune it down to 0.05 but they cant defend themselves much
Or mb its best for another follow up PR

Comment (Saugia) [reply data/governments.txt:1100]: Follow up.

**Comment by Amazinite** [comment]
We can create more specific interactions later where we see fit. This should just be our foundations to build off of.

**Pull request merged**